egs/wsj/s5/steps/nnet2/convert_lda_to_raw.sh

```diff
@@ -44,6 +44,9 @@ for f in $src/final.mdl $src/final.mat $src/splice_opts $src/cmvn_opts; do
   [ ! -f $f ] && echo "$0: expected file $f to exist" && exit 1
 done
 
+if [ -f $src/phones.txt ]; then
+  cp $src/phones.txt $dir || exit 1;
```

Comment:
instead of an if-statement it would be more compact to do just
`cp $src/phones.txt $dir 2>/dev/null`
the `2>/dev/null` suppresses the warning.  In this case, we don't need to add `|| true` at the end because no-one has done `set -e` in this script (so the bad exit status wouldn't cause the script to exit).

, and the `|| true` will prevent an error